Example Apps/Example ObjC/ButtonManager.m

```diff
@@ -52,61 +50,51 @@ - (void)setTextEnabled:(BOOL)textEnabled {
 
 - (void)setImagesEnabled:(BOOL)imagesEnabled {
     _imagesEnabled = imagesEnabled;
-
-    SDLSoftButtonObject *object = [self.sdlManager.screenManager softButtonObjectNamed:AlertSoftButton];
-    [object transitionToNextState];
-
     if (self.refreshUIHandler == nil) { return; }
     self.refreshUIHandler();
 }
 
-- (void)setToggleEnabled:(BOOL)toggleEnabled {
-    _toggleEnabled = toggleEnabled;
-    SDLSoftButtonObject *object = [self.sdlManager.screenManager softButtonObjectNamed:ToggleSoftButton];
-    [object transitionToStateNamed:(toggleEnabled ? ToggleSoftButtonImageOnState : ToggleSoftButtonImageOffState)];
-}
-
 #pragma mark - Custom Soft Buttons
 
 - (NSArray<SDLSoftButtonObject *> *)allScreenSoftButtons {
-    return @[[self sdlex_softButtonAlertWithManager:self.sdlManager], [self sdlex_softButtonToggleWithManager:self.sdlManager], [self sdlex_softButtonTextVisibleWithManager:self.sdlManager], [self sdlex_softButtonImagesVisibleWithManager:self.sdlManager]];
+    return @[[self sdlex_softButtonAlert], [self sdlex_softButtonSubtleAlert], [self sdlex_softButtonTextVisible], [self sdlex_softButtonImagesVisible]];
 }
 
-- (SDLSoftButtonObject *)sdlex_softButtonAlertWithManager:(SDLManager *)manager {
+/// Returns a soft button that shows an alert when tapped.
+/// @returns A SDLSoftButtonObject object
+- (SDLSoftButtonObject *)sdlex_softButtonAlert {
```

Comment:
The icon here is different than the swift version, additionally, it shows both the icon and the text simultaneously, while the swift version only shows the icon.

Reply:
Fixed.